packages/nextra/src/loader.ts

```diff
@@ -169,10 +169,11 @@ export default MDXContent`.trimStart()
   const themeConfigImport = themeConfig
     ? `import __nextra_themeConfig__ from '${slash(path.resolve(themeConfig))}'`
     : ''
+
   const pageOpts: Omit<PageOpts, 'title'> = {
-    filename,
+    filePath: path.relative(process.cwd(), resourcePath),
```

Comment:
This fixes the path for Windows:

```suggestion
    filePath: path.relative(process.cwd(), resourcePath).replace(/\\/g, '/'),
```

Reply:
Thanks, I think we can use `slash` package instead